packages/flutter/lib/src/material/text_field.dart

```diff
@@ -92,6 +92,28 @@ class TextField extends StatefulWidget {
   /// [maxLengthEnforced], [scrollPadding], [maxLines], [maxLength],
   /// and [enableInteractiveSelection] arguments must not be null.
   ///
+  /// ## Sample code
+  ///
+  /// This example shows how to create a [TextField] as a password input field
+  /// which allows input of up to 25 characters and calls a callback method when
+  /// the user presses the 'done' button on the keyboard.
+  ///
+  /// ```dart
+  /// void _login(String passwd) {
+  ///   // provide login logic
+  /// }
+  ///
+  /// ...
```

Comment:
Sorry, I made a mistake, it should be this (so that the example code is evaluated properly by the compiler):
```suggestion
  /// // ...
```

Reply:
Yes, you should rebase.